### PR TITLE
Add InputSelect component

### PR DESCRIPTION
**Pull request by MathieuDebit**
image

## Usage
See its [stories](https://github.com/getstation/theme/pull/43/files#diff-809e3d1d88d3db6cfcd4e2d9cc759c30R11).

## Todo
- [x] Changelog
- [x] Lint
- [x] Impove typing

### CODE REVIEW

**Comment by MathieuDebit** [comment]
7e5e413 published as `0.15.0-rc2`.

**Comment by MathieuDebit** [comment]
11f4b79 published as `0.15.0-rc3`

File: lib/components/RoundPicture/index.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  item,
+  style,
+}: Props) => (
+  <div className={classNames(classes.container, className)} style={style}>
+    <img className={classes!.picture} src={item.picture} alt={item.name} />
+  </div>
+);
+
+// @ts-ignore
+export const RoundPicture = injectSheet(styles)(RoundPictureImpl);
```
Comment (alexstrat) [inline lib/components/RoundPicture/index.tsx]: export typing

File: lib/components/SelectInput/index.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        components={{
+          Option: this.renderOption,
+          NoOptionsMessage: this.renderNoOptionsMessage,
+        }}
+      />
+    );
+  }
+}
+
+export const SelectInput = injectSheet(styles as IgnoreJSSNested<typeof styles>)(SelectInputImpl);
```
Comment (alexstrat) [inline lib/components/SelectInput/index.tsx]: export typing

**Pull request merged**